package.json

```diff
@@ -32,51 +32,47 @@
     "prepublishOnly": "yarn build"
   },
   "dependencies": {
-    "@hoprnet/hopr-utils": "1.74.0-next.16",
+    "@hoprnet/hopr-utils": "1.75.0-next.2",
     "abortable-iterator": "^3.0.0",
     "bl": "^5.0",
-    "debug": "^4.3.1",
-    "heap-js": "^2.1.5",
+    "debug": "^4.3.2",
+    "heap-js": "^2.1.6",
     "it-handshake": "^2.0.0",
-    "multiaddr": "~9.0.2",
+    "multiaddr": "^10.0.0",
     "multihashes": "^4.0.2",
     "p-defer": "3",
-    "peer-id": "~0.14.8",
+    "peer-id": "~0.15.1",
     "simple-peer": "9.11",
-    "stream-to-it": "^0.2.3",
+    "stream-to-it": "^0.2.4",
     "webrtc-stun": "3.0.0",
     "wrtc": "0.4.7"
   },
   "devDependencies": {
     "@types/debug": "^4.1.6",
-    "@types/mocha": "^8.2.2",
+    "@types/mocha": "^8.2.3",
     "@types/node": "14",
     "@types/simple-peer": "^9.11",
     "@types/yargs": "^17.0.2",
     "abort-controller": "3.0.0",
     "it-pair": "^1.0.0",
     "it-pipe": "^1.1.0",
     "it-pushable": "^1.4.2",
-    "libp2p": "^0.31.6",
-    "libp2p-interfaces": "^0.11",
+    "libp2p": "^0.32.0",
```

Comment:
Why is libp2p only a devDependency?

Reply:
Libp2p is only used for integration tests